magellan-legacy/src/main/java/com/wealthfront/magellan/Navigator.kt

```diff
@@ -14,17 +15,17 @@ import com.wealthfront.magellan.transitions.DefaultTransition
 import com.wealthfront.magellan.transitions.MagellanTransition
 import com.wealthfront.magellan.transitions.NoAnimationTransition
 import com.wealthfront.magellan.transitions.ShowTransition
-import java.lang.IllegalStateException
-import java.util.Stack
+import rewriteHistoryWithNavigationEvents
+import java.util.Deque
 
-public class LegacyNavigator internal constructor(
+public class Navigator internal constructor(
```

Comment:
What was the thinking behind renaming this again?

Reply:
So i wanted to keep the integration simple so this should help so that I don't have to change all the places in the app that imports the navigator for now. But I will probably change it back in a small update after the integration.